packages/core/src/browser/style/tabs.css

```diff
@@ -320,6 +320,14 @@ body.theia-editor-highlightModifiedTabs
   align-items: center;
 }
 
+.p-TabBar-tab.drop-target-top {
+  border-top: 2px solid white !important;
+}
+
+.p-TabBar-tab.drop-target-bottom {
+  border-bottom: 2px solid white !important;
+}
+
```

Comment:
I think the problem still exists. I think we probably need to set the `box-sizing` property for these tabs to be `border-box` (hopefully that will fix it - it may be more complex) so that they retain the same total size regardless of the presence or absence of the border.

To test, grab a view container part, and then drag up and down the sidebar over the various other tabs. The borders for the tabs should appear and disappear, but the icons should remain in the same position.